survey-sdk/build.gradle

```diff
@@ -53,3 +54,20 @@ dependencies {
         "androidx.test.ext:junit:$junit_androidx_version"
     )
 }
+
+afterEvaluate {
+    publishing {
+        publications {
+            // Creates a Maven publication called "release".
+            release(MavenPublication) {
+                // Applies the component for the release build variant.
+                from components.release
+
+                // You can then customize attributes of the publication as shown below.
+                groupId = 'co.nimblehq'
+                artifactId = 'survey.sdk'
+                version = '0.0.2'
```

Comment:
version conflicts to 0.1.0 here https://github.com/nimblehq/survey-api-sdk-android/pull/31/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R29

Reply:
@luongvo Yes, let me correct it, it should be 0.1.0

Reply:
@luongvo Updated